DataFormats/BeamSpotAlpaka/interface/alpaka/BeamSpotAlpaka.h

```diff
@@ -0,0 +1,36 @@
+#ifndef DataFormats_BeamSpotAlpaka_interface_alpaka_BeamSpotAlpaka_h
+#define DataFormats_BeamSpotAlpaka_interface_alpaka_BeamSpotAlpaka_h
+
+#include <alpaka/alpaka.hpp>
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+#include "DataFormats/BeamSpot/interface/BeamSpotPOD.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+  class BeamSpotAlpaka {
```

Comment:
I would prefer to avoid using `Alpaka` in the name of the types and of the packages.